src/constant.ts

```diff
@@ -10,12 +10,19 @@ export const FT_MINIMUM_STORAGE_BALANCE_LARGE =  "12500000000000000000000"
 export const ONE_YOCTO_NEAR = '0.000000000000000000000001';
 let ENV: string | undefined = '';
 let INDEXER_URL: string | undefined = '';
+let CONFIG: Config | undefined = undefined;
```

Comment:
Why do you need this for?

Reply:
I thought it should be like the INDEXER_URL, changed.

Reply:
ok this is needed so it can get no value